### PR TITLE
Blog for release of v1.2.0

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: content/post/release-of-1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+date: "2017-10-12T16:21:30+08:00"
+author: "lunny"
+title: "Release of 1.2.0"
+tags: ["release"]
+draft: false
+---
+
+We proudly present that we have released the third big version of Gitea v1.2.0. We have merged [676](https://github.com/go-gitea/gitea/milestone/3?closed=1) pull requests to release this version. You can download one of our pre-built binaries from our [downloads page](https://dl.gitea.io/gitea/1.2.0/), you just need to select the correct platform. For further details of the installation follow our [installation guide](https://docs.gitea.io/en-us/install-from-binary/).
+
+This version is delayed about six months and it's not obey our release cycle plan. We changed the Logo for a new design by voting. Many new features added like Status, 20+ OAuth2 and OpenID login, Team Units, GPG validation, Batch operation of issues and etc. Also we fixed about 200+ bugs on this release cycle and it spent our many time. For next release [v1.3.0](https://github.com/go-gitea/gitea/milestone/9), I wish we can reduce the PRs and release ASAP!
```
Comment (lafriks) [inline content/post/release-of-1.2.0.md]: Should be `... was delayed for six month but it should not obey our ...`

Comment (sondr3) [reply content/post/release-of-1.2.0.md]: The whole second paragraph reads a bit weird, I think it would benefit from some rewriting 😃

Comment (lafriks) [reply content/post/release-of-1.2.0.md]: @sondr3 feel free to suggest ;)

**Comment by strk** [comment]
LGTM (to unlblock this, I can't stand the excessive blocking of flow here)

**Comment by lafriks** [comment]
LGTM

**Pull request merged**